libctru/source/services/fs.c

```diff
@@ -9,20 +9,30 @@
 #include <3ds/env.h>
 #include "../internal.h"
 
+#define FS_MAX_ARCHIVE_HANDLES 32
```

Comment:
But what IF? Then ctrulib needs to be changed again. I would make a compromise between size and usability like 15 handles.